tempodb/encoding/vparquet/block_iterator.go

```diff
@@ -28,7 +28,7 @@ func (b *backendBlock) open(ctx context.Context) (*parquet.File, *parquet.Reader
 	return pf, r, nil
 }
 
-func (b *backendBlock) RawIterator(ctx context.Context, pool *rowPool) (*rawIterator, error) {
+func (b *backendBlock) rawIter(ctx context.Context, pool *rowPool) (*rawIterator, error) {
```

Comment:
why did we change this?

Reply:
It was complaining that an exported function was returning and unexported type. And we don't need the function to be exported.